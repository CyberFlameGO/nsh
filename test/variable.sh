a=123
b=456

echo $a
echo $b
echo ${undefined}
echo ${undefined-}
echo ${undefined-hello}
echo ${undefined:-0${a}45}
