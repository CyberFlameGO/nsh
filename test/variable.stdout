123
456


hello
012345
